packages/components/src/List/Toolbar/Toolbar.scss

```diff
@@ -1,12 +1,14 @@
 // Toolbar styles
 $tc-list-toolbar-display-color: $silver !default;
-$tc-list-toolbar-background-color: $concrete !default;
+$tc-list-toolbar-background-color: $wild-sand !default;
+$tc-list-toolbar-height: 5.83rem !default;
 
 .tc-list-toolbar {
 	background: $tc-list-toolbar-background-color;
 	border-top: 1px solid $alto;
 	border-bottom: 1px solid $alto;
 	margin-bottom: 0;
+	padding-left: $padding-normal;
```

Comment:
The padding is composed by the action bar padding and this padding.
Let's not do that, remove the padding for the actionbar, and set a proper padding here ($padding-larger)